public/css/widget/migrate-popup.less

```diff
@@ -8,6 +8,8 @@
   top: 0;
   right: 4em;
   pointer-events: none;
+  line-height: 1.5em;
+  font-size: .875em;
```

Comment:
Why is that on the entire popup? This way it has an effect not only on font-size, but on the entire layout of the popup. I thought it's only the text that's too big.

Currently I have difficulty too read the "Don't show this again" label. It's way too small.